Api/V8/Config/Services/beanAliases.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+return [
```

Comment:
The file path "Config/Servers ..." - this suggests that it is a namespace. However there are no namespaces declared here.

Reply:
Why should I set a namespace for requiring a single file, which returns an array?!